src/test/java/eu/fasten/maven/CheckMojoIT_noPkgCollision.java

```diff
@@ -33,8 +33,9 @@
  * 
  * @version $Id$
  */
+// TODO maven was complaining about a collision with a Package name this should be addressed
```

Comment:
Maven is not complaining AFAIK, but Eclipse is indeed not a fan of it, I reported https://github.com/khmarbaise/maven-it-extension/issues/178#issuecomment-1002624558. Unfortunately, there is no much I can do (other than using a completely different tool of course).